rpcserver.go

```diff
@@ -2129,10 +2173,17 @@ func (r *rpcServer) parseOpenChannelReq(in *lnrpc.OpenChannelRequest,
 
 	// Skip estimating fee rate for PSBT funding.
 	if in.FundingShim == nil || in.FundingShim.GetPsbtShim() == nil {
+		// Keep the old behavior prior to 0.18.0 - when the user
+		// doesn't set fee rate or conf target, the default conf target
+		// of 6 is used.
+		targetConf := maybeUseDefaultConf(
```

Comment:
Perhaps we should just push this logic into `CalculateFeeRate`? Then we just need to remember to use that function, instead of sprinkling this additional function call everywhere.

Reply:
yeah initially did that then realized there's no logging in that package so took the easy path😂 let me know if you wanna enable logging in `lnrpc`.

Reply:
Ahh lol, yeah np to add logging here IMO.